test/models/post_image_manager_test.rb

```diff
@@ -4,6 +4,7 @@
 class PostImageManagerTest < ActiveSupport::TestCase
   test 'add_file should create a new PostImageUploader and cache the file' do 
     # Arrange
+    setup_clear_mocks
```

Comment:
Instead of adding this step before the rest of the steps in every test, can we add it to the suite's test setup?

Reply:
Yep, will do.

Reply:
Alright, that test class should have a setup method now.

Reply:
Awesome, thanks!